Makefile.include

```diff
@@ -288,9 +288,12 @@ endif
 # which is not needed in a production environment but helps in the
 # development process:
 DEVELHELP ?= 0
-ifeq ($(DEVELHELP),1)
+ifneq (0, $(DEVELHELP))
   CFLAGS += -DDEVELHELP
 endif
+ifeq (2, $(DEVELHELP))
+  CFLAGS += -DDEBUG_ASSERT_VERBOSE
+endif
```

Comment:
How about as a first step add

```suggestion
DEBUG_ASSERT_VERBOSE ?=0
ifeq ($(DEBUG_ASSERT_VERBOSE),1)
  CFLAGS += -DDEBUG_ASSERT_VERBOSE
endif
```

Reply:
That would already be quite helpful. Maybe even a shorter name, e.g. `VERBOSE_ASSERT`  or `ASSERT_VERBOSE`, as this is often typed by hand?